clients/js/src/ChromaClient.ts

```diff
@@ -110,7 +113,8 @@ export class ChromaClient {
    * const version = await client.version();
    * ```
    */
-  public async version(): Promise<string> {
+  async version(): Promise<string> {
+    await this.initPromise;
```

Comment:
These calls previously did not require auth but I'd argue that, given that the Chroma Client is scoped to a tenant and database, it's reasonable to have these fail if the tenant and database don't exist or cannot be accessed. But fine to revert if we want to keep these accessible. Also, potentially these should live on the admin interface, in that case?

Reply:
> Also, potentially these should live on the admin interface, in that case?

having `.version()` at least accessible on the normal client seems fine